Dockerfile.test.php7.debug

```diff
@@ -1,9 +1,9 @@
-FROM php:7.0
+FROM php:7.2
```

Comment:
Running this file (`npm run tests:unit:debug`) fails without these changes.

Reply:
This made me nervous until I checked the .travis.yml and you're right, it's doing

`- sed -i "s/7.2/$TRAVIS_PHP_VERSION/g" Dockerfile.test.php7` (7.2, not 7.0).. so how is this working now?